cpp_test_suite/new_tests/cxx_old_poll.cpp

```diff
@@ -880,7 +880,7 @@ class OldPollTestSuite__loop : public CxxTest::TestSuite {
             cout << endl;
         }
 
-        auto nb_polled = BASIC_NB_POLL + 1;
+        std::size_t nb_polled = BASIC_NB_POLL + 1;
```

Comment:
Is it necessary? I would prefer auto and the latest c++ features in tests. IMHO It is ok, as tests ain't part of the distribution

Reply:
If we use auto here, nb_polled will become an int.
And then there is a compilation warning:
```
/home/tango/src/cpp_test_suite/cxxtest/include/../../new_tests/cxx_old_poll.cpp:814:9: note: in expansion of macro 'TS_ASSERT'
         TS_ASSERT(v_str.size() == nb_polled);
         ^~~~~~~~~
/home/tango/src/cpp_test_suite/cxxtest/include/../../new_tests/cxx_old_poll.cpp: In member function 'void OldPollTestSuite__loop::test_poll_command()':
/home/tango/src/cpp_test_suite/cxxtest/include/../../new_tests/cxx_old_poll.cpp:893:36: warning: comparison of integer expressions of different signedness: 'std::vector<std::__cxx11::basic_string<char> >::size_type' {aka 'long unsigned int'} and 'int' [-Wsign-compare]
         TS_ASSERT(poll_str->size() == nb_polled);
                   ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
```